### PR TITLE
[new] MST REST API

### PR DESCRIPTION
**Pull request by Rajakavitha1**
# Description

Moved the content from this [article](https://app.intercom.com/a/apps/uvcadzno/articles/articles/2949775/show) to MST docs

# Links

Fixes #[insert issue link, if any]

# Writing help

For information about style and word usage, see the [style guide](https://docs.timescale.com/timescaledb/latest/contribute-to-docs/)

# Review checklists
Reviewers: use this section to ensure you have checked everything before approving this PR:

## Subject matter expert (SME) review checklist

- [ ] Is the content technically accurate?
- [ ] Is the content complete?
- [ ] Is the content presented in a logical order?
- [ ] Does the content use appropriate names for features and products?
- [ ] Does the content provide relevant links to further information?

## Documentation team review checklist

- [ ] Is the content free from typos?
- [ ] Does the content use plain English?
- [ ] Does the content contain clear sections for concepts, tasks, and references?
- [ ] Have any images been uploaded to the correct location, and are resolvable?
- [ ] If the page index was updated, are redirects required
      and have they been implemented?
- [ ] Have you checked the built version of this content?

### CODE REVIEW

**Review by charislam — CHANGES_REQUESTED** [review]
Hey Rajie, do you mind adding the page-index entry? I think something is wrong with the indentation but I can't double-check because it won't show up in preview without a page-index entry

**Comment by Rajakavitha1** [comment]
> Hey Rajie, do you mind adding the page-index entry? I think something is wrong with the indentation but I can't double-check because it won't show up in preview without a page-index entry

Thanks @charislam !!! I just updated the page-index entry

**Review by charislam — APPROVED** [review]
Yep the indentation was off!

If you want code blocks to appear as part of the same ordered list, rather than breaking the ordered list, they have to be indented at the same level.

Just pushed a fix and it should be good now!

**Comment by Rajakavitha1** [comment]
> Yep the indentation was off!
> 
> If you want code blocks to appear as part of the same ordered list, rather than breaking the ordered list, they have to be indented at the same level.
> 
> Just pushed a fix and it should be good now!

Thank you @charislam !!!

File: mst/restapi.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+title: Using REST API in Managed Service for TimescaleDB
+excerpt: Use REST API in Managed Service for TimescaleDB for integration and automation
+product: mst
+keywords: [rest api]
+---
+
+# Using REST API in Managed Service for TimescaleDB
+
+Managed Service for TimescaleDB has an API for integration and automation tasks.
+For information about using the endpoints, see the [API Documentation](https://api.aiven.io/doc/).
```
Comment (Loquacity) [inline mst/restapi.md:11]: Turn this into a reference link.

File: mst/restapi.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    token and leave the rest of the fields blank.
+1.  Copy the generated authentication token and save it.
+
+</procedure>
+
+<procedure>
+
+### Using cURL to get your details
+
+1.  Set the environmental variable `MST_API_TOKEN` with the access token that you generated.
```
Comment (Loquacity) [inline mst/restapi.md:33]:
```suggestion
1.  Set the environment variable `MST_API_TOKEN` with the access token that you generate:
```

File: mst/restapi.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+    export MST_API_TOKEN="access token"
+    ```
+
+1.  To get the details about the current user session using the `/me` endpoint:
+
+    ```bash
+    curl -s -H "Authorization: aivenv1 $MST_API_TOKEN" https://api.aiven.io/v1/me|json_pp
+    ```
+
+    An output similar to the following appears:
```
Comment (Loquacity) [inline mst/restapi.md:45]:
```suggestion
    The output looks similar to this:
```

File: mst/restapi.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+---
+
+# Using REST API in Managed Service for TimescaleDB
+
+Managed Service for TimescaleDB has an API for integration and automation tasks.
+For information about using the endpoints, see the [API Documentation](https://api.aiven.io/doc/).
+MST offers an HTTP API with token authentication and JSON-formatted data. You
+can use the API for all the tasks that can be performed using the web console.
+
+<procedure>
```
Comment (Loquacity) [inline mst/restapi.md:15]: This probably needs to be laid out as a process/procedure.

Comment (Rajakavitha1) [reply mst/restapi.md:17]: I added s sentence now that explains the need to create an authentication token.

File: mst/restapi.md (hunk 1 above)
Comment (Rajakavitha1) [reply mst/restapi.md]: done

**Review by Loquacity — APPROVED** [review]

**Review by jfjoly — APPROVED** [review]
LGTM, thanks!
I would add a last section "what's next?" pointing to the API reference guide of Aiven (https://api.aiven.io/doc/) once the user has confirmed he can query the API. This can be added later.

**Pull request merged**